tests/virt_autotest/install_package.pm

```diff
@@ -107,17 +107,14 @@ sub install_package {
 }
 
 sub run {
-    if (is_x86_64) {
-        script_run("zypper ar -p 80 http://download.suse.de/ibs/home:/Julie_CAO:/branches:/workaround/SLE-15-SP6 workaround");
-        script_run("zypper --gpg-auto-import-keys ref -r workaround");
-        save_screenshot;
-    }
+    script_run("zypper ar -p 80 http://download.suse.de/ibs/home:/Julie_CAO/SLE-15-SP6 julie");
+    script_run("zypper --gpg-auto-import-keys ref -r julie");
+    save_screenshot;
     install_package;
-    if (is_x86_64) {
-        script_run("zypper info qa_lib_virtauto");
-        script_run("zypper info qa_lib_keys");
-        save_screenshot;
-    }
+    script_run("zypper info qa_lib_virtauto");
```

Comment:
Why check with the following pkg info?

Meanwhile, Try to the following way: 
`script_run("zypper info qa_lib_virtauto qa_tools qa_lib_keys ");`

Reply:
all the changes in `install_package.pm` are workaround and checkpoint. There will be no any change to this file finally.